pubsub/subscription.go

```diff
@@ -489,6 +489,12 @@ type ReceiveSettings struct {
 	// for unprocessed messages.
 	MaxOutstandingBytes int
 
+	// if UseLegacyFlowControl is true, then enforcing flow control settings at
```

Comment:
Exported fields in Go should start with the field name, e.g. "UseLegacyFlowControl disables flow control settings on the Cloud Pub/Sub server and ...."

Reply:
done.